workshops/views.py

```diff
@@ -375,6 +375,61 @@ def person_bulk_add_confirmation(request):
         return render(request, 'workshops/person_bulk_add_results.html',
                       context)
 
+@login_required
+def person_find_duplicates(request):
```

Comment:
I have a couple of notes regarding this function.

1) Please follow [PEP8](https://www.python.org/dev/peps/pep-0008/) especially when it comes to line length
2) ...and to spaces around colons and commas and `>`
3) You duplicate about 9 lines of code in lines 397 and 432 - it's against Don't Repeat Yourself rule
4) There's no handling for methods other than `POST` and `GET` (I'm nitpicking here.)